src/templates/BillOfLading/sampleV3.ts

```diff
@@ -0,0 +1,61 @@
+import { v3 } from "@govtechsg/open-attestation";
+import { BillOfLadingSchemaV3 } from "./types";
+
+export const BillOfLadingSampleV3: BillOfLadingSchemaV3 = {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "http://localhost:3000/BOL",
+    "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
```

Comment:
```suggestion
    "https://www.w3.org/2018/credentials/v1",
    "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
    "https://schemata.openattestation.com/io/tradetrust/bill-of-lading/1.0/bill-of-lading-data.json",
```

Reply:
done, changed it to https://schemata.openattestation.com/io/tradetrust/bill-of-lading/1.0/bill-of-lading-context.json